apis/objectstorage.k8s.io/v1alpha1/types.go

```diff
@@ -118,7 +118,7 @@ type BucketRequest struct {
 
 type BucketRequestSpec struct {
 	// +optional
-	BucketInstanceName string `json:"bucketInstanceName,omitempty"`
+	BucketName string `json:"bucketName,omitempty"`
```

Comment:
this field is no longer in BR.spec (per latest kep pr). Instead, following api guidance, `bucketName` is now defined in BR.status.

Reply:
got it